spec/requests/api/v1/trails/index_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'rails_helper'
+
+RSpec.describe 'GET /trails' do
+    describe "trails index" do
+        it " returns all trails " do
+            json_response = File.read('spec/fixtures/DO_NOT_DELETE/trails.json')
```

Comment:
What is the purpose of this stubbed data file here? Keep in mind that in a request spec, we do not want to stub the request but we want RSpec to make a request to an endpoint (like you've set up on line 9) and make assertions on the response.

Reply:
If you want seeded data for this test so that the response actually returns trails, consider using faker/factory bot instead to set up some test data (instead of using a stubbed request)

Reply:
I was having testing issues when working on the show_spec without the stub data. So I kind of just jumped the gun and assumed that this would also need the stub data. Testing it locally now i realize it is not required will update this!

Reply:
Sounds good! Keep in mind you will need test data in order for anything to show up, but you can do that with `Trails.create` or using factoryBot